packages/cache/src/useSuspenseCache.spec.tsx

```diff
@@ -0,0 +1,113 @@
+import { ErrorBoundary, Suspense } from '@suspensive/react'
+import { ERROR_MESSAGE, FALLBACK, TEXT, sleep } from '@suspensive/test-utils'
+import { render, screen, waitFor } from '@testing-library/react'
+import ms from 'ms'
+import { Cache } from './Cache'
+import { CacheProvider } from './CacheProvider'
+import { useSuspenseCache } from './useSuspenseCache'
+
+const key = (id: number) => ['key', id] as const
+
+const SuspenseCacheSuccess = () => {
+  const resolvedData = useSuspenseCache({ cacheKey: key(1), cacheFn: () => sleep(ms('0.1s')).then(() => TEXT) })
+
+  return (
+    <>
+      {resolvedData.data}
+      <button onClick={resolvedData.reset}>Try again</button>
+    </>
+  )
+}
+
+const SuspenseCacheFailure = () => {
+  const resolvedData = useSuspenseCache({
```

Comment:
Use cacheOptions please

Reply:
I modified it https://github.com/toss/suspensive/pull/1083/commits/ceb14e435a346cfb2f3a4ea546dc60bb7444a34a